test/test_multiprocessing.py

```diff
@@ -222,6 +230,7 @@ def test_receive():
             t2 = q.get()
             self.assertTrue(t1.eq(1).all())
             self.assertTrue(id(t1.storage()) == id(t2.storage()))
+            del t1, t2
```

Comment:
Oh interesting. This is because the child process is kept live if you don't delete the tensors from the receiving process, is that right? Seems worth a comment :)

Reply:
Almost right. Child process will terminate, but LOG(warning) going to be displayed.